bower.json

```diff
@@ -6,11 +6,21 @@
     "polymer"
   ],
   "dependencies": {
-    "iron-ajax": "^1.4.3",
-    "polymer": "^1.8.0",
+    "iron-ajax": "^2.0.7",
+    "polymer": "1 - 2",
     "base64-js": "^1.2.1"
   },
   "devDependencies": {
     "web-component-tester": "^6.0.0"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "^1.9.1"
+      },
+      "resolutions": {
+        "webcomponentsjs": "^0.7"
+      }
+    }
   }
```

Comment:
Based on other files I've seen, does this also need a `resolution` for `webcomponentsjs: ^1.0.0`?